cucumber-messages/messages.proto

```diff
@@ -433,31 +458,32 @@ message TestStepMatched {
 message TestStepStarted {
   string pickleId = 1;
   uint32 index = 2;
-  google.protobuf.Timestamp timestamp = 3;
+  Timestamp timestamp = 3;
 }
 
 message TestStepFinished {
   string pickleId = 1;
   uint32 index = 2;
   TestResult testResult = 3;
-  google.protobuf.Timestamp timestamp = 4;
+  Timestamp timestamp = 4;
 }
 
 message TestHookStarted {
   string pickleId = 1;
-  google.protobuf.Timestamp timestamp = 2;
+  Timestamp timestamp = 2;
 }
 
 message TestHookFinished {
   string pickleId = 1;
   TestResult testResult = 2;
-  google.protobuf.Timestamp timestamp = 3;
+  Timestamp timestamp = 3;
 }
 
 message TestResult {
   Status status = 1;
   string message = 2;
-  uint64 durationNanoseconds = 3;
+  uint64 durationNanoseconds = 3 [deprecated=true];
```

Comment:
Up until now we have not used protobuf's deprecated feature. Instead we are simply replacing the field with `reserved`. See the comment on line 7.

Once we have stuff in production we can consider `deprecated`, but for now I think it is best to remove it (more thorough cleanup, and no risks).

Reply:
Works for me. I wasn't sure about Specflow handling of messages, but I think we can discuss it in the coming meeting with them.
I thought the deprecation would make it easier on their side :)

Reply:
I think deprecation will only make it easier to keep using the old field, which we want to avoid.

Reply:
that's removed